docs/book/component-guide/orchestrators/databricks.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Orchestrating your pipelines to run on Databricks.
+---
+
+# Databricks Orchestrator
+
+[Databricks](https://www.databricks.com/) is a unified data analytics platform that combines the best of data warehouses and data lakes to offer an integrated solution for big data processing and machine learning. It provides a collaborative environment for data scientists, data engineers, and business analysts to work together on data projects. Databricks is built on top of Apache Spark, offering optimized performance and scalability for big data workloads.
+
+The Databricks orchestrator is an orchestrator flavor provided by the ZenML databricks integration that allows you to run your pipelines on Databricks. This integration enables you to leverage Databricks' powerful distributed computing capabilities and optimized environment for your ML pipelines within the ZenML framework.
+
+{% hint style="warning" %}
+This component is only meant to be used within the context of a [remote ZenML deployment scenario](../../getting-started/deploying-zenml/README.md). Usage with a local ZenML deployment may lead to unexpected behavior!
+{% endhint %}
+
+### When to use it
+
+You should use the Databricks orchestrator if:
+
+* you're already using Databricks for your data and ML workloads.
+* you want to leverage Databricks' powerful distributed computing capabilities for your ML pipelines.
+* you're looking for a managed solution that integrates well with other Databricks services.
+* you want to take advantage of Databricks' optimization for big data processing and machine learning.
+
+### Prerequisites
+
+You will need to do the following to start using the Databricks orchestrator:
+
+* An Active Databricks workspace, depends on the cloud provider you are using, you can find more information on how to create a workspace:
+    * [AWS](https://docs.databricks.com/en/getting-started/onboarding-account.html)
+    * [Azure](https://learn.microsoft.com/en-us/azure/databricks/getting-started/#--create-an-azure-databricks-workspace)
+    * [GCP](https://docs.gcp.databricks.com/en/getting-started/index.html)
+* Active Databricks account or service account with sufficient permission to create and run jobs
+
+## How it works
+
+The Databricks orchestrator in ZenML leverages the concept of Wheel Packages. When you run a pipeline with the Databricks orchestrator, ZenML creates a Python wheel package from your project. This wheel package contains all the necessary code and dependencies for your pipeline.
+
+Once the wheel package is created, ZenML uploads it to Databricks. ZenML leverage Databricks SDK to create a job definition, This job definition includes information about the pipeline steps and ensures that each step is executed only after its upstream steps have successfully completed.
```

Comment:
a diagram here would be nice, also for marketing . maybe the flow of what happens.

Reply:
Good point will create one